<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-RJ5O4JI" failures="4" tests="27" name="Default test" time="84.146" errors="0" timestamp="2023-02-07T22:36:03 EST">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.757"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.869"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.133"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.017"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.148"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.597"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.732"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.48"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.131"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.632"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.371"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.244"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.901"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="2.219"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.518"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.553"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.022"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.893">
    <failure type="java.lang.AssertionError" message="User gets submission message expected [submission success] but found [ Some Tests failed. Please review code]">
      <![CDATA[java.lang.AssertionError: User gets submission message expected [submission success] but found [ Some Tests failed. Please review code]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at stepdefinition.ArrayPage_SD.user_should_get_success_submission_message(ArrayPage_SD.java:265)
at ✽.User should get success submission message(file:///C:/Users/arunm/git/ds_algo_ninjaricos/src/test/resources/Features/TC05_Array.feature:163)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.824"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.724"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.174">
    <failure type="java.lang.AssertionError" message="User gets submission message expected [submission success] but found [ Some Tests failed. Please review code]">
      <![CDATA[java.lang.AssertionError: User gets submission message expected [submission success] but found [ Some Tests failed. Please review code]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at stepdefinition.ArrayPage_SD.user_should_get_success_submission_message(ArrayPage_SD.java:265)
at ✽.User should get success submission message(file:///C:/Users/arunm/git/ds_algo_ninjaricos/src/test/resources/Features/TC05_Array.feature:189)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.546"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.088"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.82">
    <failure type="java.lang.AssertionError" message="User gets submission message expected [submission success] but found [Error occurred during submission]">
      <![CDATA[java.lang.AssertionError: User gets submission message expected [submission success] but found [Error occurred during submission]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at stepdefinition.ArrayPage_SD.user_should_get_success_submission_message(ArrayPage_SD.java:265)
at ✽.User should get success submission message(file:///C:/Users/arunm/git/ds_algo_ninjaricos/src/test/resources/Features/TC05_Array.feature:215)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.665"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.243"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="3.913">
    <failure type="java.lang.AssertionError" message="User gets submission message expected [submission success] but found [ No tests were collected]">
      <![CDATA[java.lang.AssertionError: User gets submission message expected [submission success] but found [ No tests were collected]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at stepdefinition.ArrayPage_SD.user_should_get_success_submission_message(ArrayPage_SD.java:265)
at ✽.User should get success submission message(file:///C:/Users/arunm/git/ds_algo_ninjaricos/src/test/resources/Features/TC05_Array.feature:241)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
